code/datums/supply_packs/food.dm

```diff
@@ -84,13 +213,34 @@
 					/obj/item/reagent_container/food/snacks/grown/wheat,
 					/obj/item/reagent_container/food/snacks/grown/wheat,
 					)
-	cost = 10
+	cost = 7
 	containertype = /obj/structure/closet/crate/freezer
 	containername = "\improper Wheat crate"
 	group = "Food"
 
+/datum/supply_packs/exoticplants
+	name = "exoticplants"
+	contains = list(
+					/obj/item/grown/nettle,
+					/obj/item/reagent_container/food/snacks/grown/icepepper,
+					/obj/item/reagent_container/food/snacks/monkeycube,
+					/obj/item/reagent_container/food/snacks/grown/goldapple,
+					/obj/item/reagent_container/food/snacks/grown/ambrosiadeus,
+					/obj/item/reagent_container/food/snacks/grown/ambrosiavulgaris,
+					/obj/item/reagent_container/food/snacks/grown/mushroom/libertycap,
+					/obj/item/reagent_container/food/snacks/grown/mushroom/amanita,
+					/obj/item/reagent_container/food/snacks/grown/mushroom/plumphelmet,
```

Comment:
these can be used to cause laggy mob spam, remove

Reply:
i removed ambrosiavulgaris